cypress/e2e/pages/reviewApplicant.cy.ts

```diff
@@ -13,10 +13,7 @@ describe('Review applicant details', () => {
     cy.clearAllCookies();
     cy.loginAsUser('readOnly');
     cy.task('clearNock');
-    ReviewApplicantPage.mockHousingRegisterApiGetApplications(
-      applicationId,
-      application
-    );
+    cy.mockHousingRegisterApiGetApplications(applicationId, application);
```

Comment:
Has this been made a cypress command elsewhere?

Reply:
Yeah all in cypress/support/commands